sunbeam-python/sunbeam/commands/juju.py

```diff
@@ -1399,3 +1399,30 @@ def run(self, status: Optional["Status"] = None) -> Result:
             return Result(ResultType.COMPLETED)
         except Exception as e:
             return Result(ResultType.FAILED, str(e))
+
+
+class UpdateJujuModelConfigStep(BaseStep):
```

Comment:
How does it work with the terraform provider ? 

If the config is not present in the state, it will want to remove them.
I know at the moment it does not unset config values, but that's a bug in the provider

Reply:
+1 will change the logic to use juju terraform provider

Reply:
Modified as per your suggestion